tests/test_configargparse.py

```diff
@@ -8,17 +8,13 @@
 import tempfile
 import types
 import unittest
+from io import StringIO
 
 try:
     import mock
 except ImportError:
     from unittest import mock
```

Comment:
[Since 3.3 `mock` is inside `unittest import mock`](https://docs.python.org/3/library/unittest.mock.html#module-unittest.mock)
```suggestion
from unittest import mock
```

Reply:
I would also remove `mock` from [setup.py](https://github.com/bw2/ConfigArgParse/blob/master/setup.py)

Reply:
yes !